features/telemetry/api/pom.xml

```diff
@@ -45,5 +50,10 @@
       <groupId>io.dropwizard.metrics</groupId>
       <artifactId>metrics-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I would like to try and maintain avoiding having knowledge of specific payloads in the Telemetry API.

I realize this may be required if we want to combine the protobuf definitions though and avoid intermediary byte arrays though. IMO it would only be worth doing this if we we're able to show some associated performance benefits.

Reply:
I currently have `netflow.proto` generated files in `flows/api` project but we can move this to `telemetry/api` ( which is the model I initially had).  I moved this to `flows/api` as I was trying to avoid conversion from `FlowMessage` to `Flow`.  I don't think it offers any performance benefits as it is just converting from one DTO to another.

Reply:
We can keep it in `flows/api` then. This dependency shouldn't be needed then no?

Reply:
yes. I still have the conversion of `FlowMessage` to `Flow` in  `Netflow5Converter`.  Once everything is moved to `FlowMessage`  it will be removed.

Reply:
Sorry, I think we can remove this dependency on `flows/api` it is only needed in adapter.

Reply:
IMO, the transport protocol stuff should go to `/features/telemetry/protocols/netflow/transport` ans it is related to netflow only.

Reply:
> IMO, the transport protocol stuff should go to `/features/telemetry/protocols/netflow/transport` ans it is related to netflow only.

You mean protobuf generated files should be moved to transport ?

Reply:
Yeah. Abd they should reside under `proto/netflow/`. This allows parser and adapter to use it.